postgres-14/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
-FROM postgres:13.4
+FROM postgres:14beta3
```

Comment:
```suggestion
FROM postgres:14rc1
```

Reply:
14rc1 is debian 11 based as well so rollbacked into beta3